pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -243,7 +242,7 @@ def generateShape(self):
         tr = QtGui.QTransform()
         tr.translate(devPos.x(), devPos.y())
         va = atan2(v.y(), v.x())
-        tr.rotate(va * 180.0 / pi)
+        tr.rotate(degrees(va))
```

Comment:
There is a `rotateRadians()` method for QTransform, would that be better to use here?